src/main/java/study/step4/Cars.java

```diff
@@ -0,0 +1,38 @@
+package study.step4;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void startRacing() {
+        for (Car car : cars) {
+            car.move();
+        }
+    }
+
+    public int getWinnerPosition() {
+        return Collections.max(cars).getPosition();
+    }
+
+    public List<Car> getWinners() {
+        int winnerPosition = getWinnerPosition();
+        List<Car> winners = cars.stream().filter(car -> isWinner(car, winnerPosition)).collect(Collectors.toList());
+
+        return winners;
```

Comment:
지역 변수 선언 없이 곧바로 반환해도 충분할 것 같습니다.
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`에 따라, 한 줄에 점을 하나만 찍어보면 어떨까요?
```suggestion
        return cars.stream()
            .filter(car -> isWinner(car, winnerPosition))
            .collect(Collectors.toList());
```

Reply:
객체지향 생활 체조 원칙의 규칙 4: 한 줄에 점을 하나만 찍는다.
지키도록 하겠습니다. : )